acis_thermal_check/acis_obs.py

```diff
@@ -306,18 +314,29 @@ def acis_filter(obsid_interval_list):
             # First check to see if this is an S3 observation
             mylog.debug(f"{eachobs['obsid']}\t{eachobs['cnt_rate']}\t"
                         f"{eachobs['app_exp']*1.0e-3}\t{eachobs['num_counts']}\t"
-                        f"{eachobs['grating']}\t{eachobs['ccds']}")
-            if eachobs["ccd_count"] <= 2:
+                        f"{eachobs['grating']}\t{eachobs['ccds']}\t"
+                        f"{eachobs['spectra_max_count']}\t{eachobs['obs_cycle']}")
+            low_ct = False
+            # "New" hot ACIS category:
+            # 1. Cycle 23 and later
+            # 2. spectra_max_count must be greater than 0
+            # 3. Start time must be after ~NOV1422 load
+            if eachobs["obs_cycle"] >= 23 and \
+                    eachobs["spectra_max_count"] > 0 and \
+                    eachobs["tstart"] > new_hot_start:
+                if eachobs["instrument"] == "ACIS-I":
+                    low_ct = eachobs["spectra_max_count"] < 1000
+                elif eachobs["instrument"] == "ACIS-S":
+                    low_ct = eachobs["spectra_max_count"] < 2000
+            elif eachobs["ccd_count"] <= 2:
+                # otherwise, fall back to "old" criteria
                 # S3 with low counts
-                low_ct_s3 = eachobs["num_counts"] < 300 and "S3" in eachobs["ccds"]
+                low_ct = eachobs["num_counts"] < 300 and "S3" in eachobs["ccds"]
```

Comment:
Shouldn't this be only for ACIS-S observations (i.e. it should *NOT* flag as hot for an I3 aimpoint obs. with S3 on?).

Reply:
Sure thing, will add the explicit check